app/src/main/java/com/github/pockethub/android/ui/issue/IssueDashboardPagerAdapter.java

```diff
@@ -26,7 +26,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static com.github.pockethub.android.core.issue.IssueFilter.*;
```

Comment:
Why was this changed?

Reply:
This does not look good at all. I will declutter this and any other static imports